scripts/fuzz/generate_decoding_fuzz_corpus.py

```
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+import json
+import pathlib
+import subprocess
+
+ROOT = pathlib.Path(__file__).resolve().parents[2]
+CORPUS = ROOT / "fuzz" / "corpus"
+
+
+def run(*args: str) -> None:
+    subprocess.run(args, cwd=ROOT, check=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)